Dockerfile

```
@@ -5,6 +5,8 @@ RUN apk --no-cache add openjdk11 --repository=http://dl-cdn.alpinelinux.org/alpi
 RUN apk --update add openjdk14-jre --repository=http://dl-cdn.alpinelinux.org/alpine/edge/testing
 RUN apk --update add openjdk15-jre --repository=http://dl-cdn.alpinelinux.org/alpine/edge/testing
 RUN apk --update add openjdk16-jre --repository=http://dl-cdn.alpinelinux.org/alpine/edge/testing
+RUN apk --update add openjdk17 --repository=http://dl-cdn.alpinelinux.org/alpine/edge/testing
```

```suggestion
RUN apk --update add openjdk17-jre --repository=http://dl-cdn.alpinelinux.org/alpine/edge/testing
```

what do you think? I already did jre but perhaps I was wrong from the start

---

My understanding is the JRE does not have compiler tools etc. For running a java app the JRE is preferred, but if we are building a repo I'd think the JDK is preferable...?

---

Yes, I agree too. Not sure why thought it worked with jre then.

Well, for alignement purposes, I will switch all of them to JDK